### PR TITLE
HAAR-1233:  Added swagger docs and removed some duplicate stubs

### PR DESCRIPTION
**Pull request by ravmoj**
WIP: Added swagger docs and removed some duplicate stubs

### CODE REVIEW

**Comment by ravmoj** [comment]
- Removed some Get and Put stubs. 
- Delete and post are WIP

File: src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt [hunk 1]
```diff
@@ -61,9 +60,9 @@ class ExternalUsersApiMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
 
-  fun stubCreateRole() {
+  fun stubCreateHttpStatus(url: String) {
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt:63]: Would the name be better as stubPost() to match the others?

Comment (ravmoj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/integration/wiremock/ExternalUsersApiMockServer.kt:63]: Thanks Karen. Updated

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**